content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -7,73 +7,86 @@ reviewers:
 - liggitt
 content_type: concept
 weight: 25
-min-kubernetes-server-version: 1.16
 ---
 
 <!-- overview -->
 
 {{< feature-state for_k8s_version="v1.22" state="stable" >}}
 
-## Introduction
+Kubernetes supports multiple appliers collaborating to manage the `.spec` and / or
```

Comment:
I don't know if that's correct, we also allow multiple appliers on `.status`

Reply:
I think that's existing wording from somewhere. What's the true statement?

Reply:
You can change all the same things with apply that you can change with a regular PUT/PATCH

Reply:
:thinking: If two controllers / API clients are collaborating on managing `.status`, they might also need write access to the resource as well as to its `status` subresource?

The important detail here is that SSA supports the “multiple co-operating clients” / blackboard pattern. I'll try to reword to avoid implying that SSA can't manage `.status`.